bindkey/bindkey.schema.json

```diff
@@ -1,7 +1,7 @@
 {
-  "$id": "EAxes7ZBFHHeWsr5vlkAx-96hhlS7U-iDxt_dEytnZYI",
+  "$id": "EPQ0xcsL__xZh7WiwMp_EVlKhieA8b_uFV87SFy44vzd",
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "title": "Bind Key",
+  "title": "Bind Key Credential",
```

Comment:
This is not a credential. It doesn't have an issuee. It is an Attestation.

Reply:
updated as per comment